conda/install.py

```diff
@@ -525,28 +542,20 @@ def move_to_trash(prefix, f, tempdir=None):
             if e1.errno != errno.EEXIST:
                 continue
 
-        if tempdir is None:
-            import tempfile
-            trash_dir = tempfile.mkdtemp(dir=trash_dir)
-        else:
-            trash_dir = join(trash_dir, tempdir)
+        trash_dir = tempfile.mkdtemp(dir=trash_dir)
 
         try:
-            try:
-                os.makedirs(join(trash_dir, dirname(f)))
```

Comment:
I guess I did this for debugging purposes (you can easily see where the trash files are coming from). I would keep this here, unless it is causing problems.